addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -95,7 +95,7 @@ def test_body_escape(self):
         )
         self.assertEqual(message.body, expected)
         ticket_record._message_update_content(message, "Hello <R&D/>")
-        self.assertEqual(message.body, Markup("<p>Hello &lt;R&amp;D/&gt;</p>"))
+        self.assertEqual(message.body, Markup('<p>Hello &lt;R&amp;D/&gt;<span class="o-mail-message-edited"></span></p>'))
```

Comment:
This result is strange, it means the parser is undoing self closing tags?
But not a big deal.